src/main/java/edu/iis/powp/command/ICompoundCommand.java

```diff
@@ -9,4 +9,5 @@
 
 	public Iterator<IPlotterCommand> iterator();
 
+	public String getName();

```

Comment:
FIX TO MERGE: getName in ICompoudCommand makes compile errors in a (recently merged) commandtrasformer package. Problem with running software. Remove getName or add support by commandtransformer.